pyproject.toml

```diff
@@ -1,5 +1,13 @@
+[project]
+name = "tedana"
+version = "0.0.12"
+
+[project.scripts]
+tedana = "tedana.workflows.tedana:_main"
+tedana_reclassify = "tedana.workflows.tedana_reclassify:_main"
```

Comment:
I generally agree, but there are now a lot of functions that will can be used more generally. I'd slightly lean towards keeping `tedana_` so that people know where this program came from, but I'd also be fine if other's prefer `ica_` Let's discuss at the next dev call.

Reply:
At our dev call, everyone was neutral or preferred `ica_reclassify` so we should change the name